### ISSUE TITLE
too many empty dayplan records

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#85]
There are so many empty dayplan entries. They all need to be deleted.
We should do that automatically somewhere. The synchronization should have something like a repair step after sync was sucessful, but that's a bigger task.

**Comment by urapadmin** [comment]
### @test recording 11.1
From now on filemaker will delete all new empty dayplans before new data gets imported into kiosk (import from filemaker step 
in the synchronization cycle).
How to test:
- create a few empty dayplans somewhere where already empty dayplans from earlier session exist
- import your workstation, sync and prepare it again. 
- [ ] In the new version only the empty dayplans you created should be missing. The ones from earlier sessions should still be there.

**Comment by urapadmin** [comment]
This recording improvement only prevents new empty dayplans from entering kiosk. To deal with existing empty dayplans we need something on kiosk level: New ticket.

**Comment by luizaogs** [comment]
Hmm, didn't work. I added 3 empty dayplans to unit MA and they are still there after uploading, synchronizing, and downloading again. And I think synchronization is telling me that records were added? It says that 3 new lines were inserted.

![Screen Shot 2021-09-19 at 8 52 52 AM](https://user-images.githubusercontent.com/67970402/133928450-afb1f8e0-fd91-41d1-89ac-d644d75e3f51.png)

**Comment by urapadmin** [comment]
hmpf

**Comment by urapadmin** [comment]
Chicago! You are an hour earlier than Providence, right? We must set your workstation to GMT-5. 
Can you do that? And then try again?

**Comment by luizaogs** [comment]
Oh sure, but I changed my computer's time to PVD before doing this so not sure that would be the issue?

**Comment by urapadmin** [comment]
Hm. That should have solved it, if you changed your time to PVD at the very beginning. 
However, the correct way with the newer versions is indeed to set your workstation itself to GMT-5.

I should tell you how:
- reset your workstation in kiosk
- now you can choose "edit workstation" under "more options"
- add your timezone GMT-5 in the timezone field (not sure what it is called, right now).

But since you already changed your computer's time I have my doubts, indeed, that things get better.

**Comment by luizaogs** [comment]
Sorry, I know how to do it but I didn't feel like calculating the GMT time at 7am :) I'll set it now, though, so that it is ready for later.

**Comment by luizaogs** [comment]
Ok, 'tis done. Let me know if you do want me to try it this way!

**Comment by urapadmin** [comment]
no, I just found the issue. It is a bit knotty and needs a new kiosk version as well. 
->Postponed for a little bit.

**Comment by urapadmin** [comment]
You think I calculate the GMT Time? Ha. I have given up on that months ago. I just google Chicago 🐑

**Comment by luizaogs** [comment]
Well that's because you're a smart person. My 7am brain didn't think of that!

**Comment by urapadmin** [comment]
the 7 modified lines in qc_flags and qc_rules are a bug. New ticket:

**Comment by urapadmin** [comment]
###@test kiosk 0.13.5 and recording 11.5
- [ ] legacy data should not contain any empty dayplans any longer (hard to test unless you remember a unit that had some)
- [ ] second try: When closing filemaker, importing and synchronizing recently added empty dayplans should disappear (so here the 11.1 tests should be repeated).

**Comment by luizaogs** [comment]
This remains fine.

**Comment by urapadmin** [comment]
- does not seem to work anymore: see #3276

**Comment by urapadmin** [comment]
@luizaogs could you please test if this still works?

**Comment by luizaogs** [comment]
I still can't add any empty dayplans - they do not survive synchronization. I don't see any old empty ones in PVD either. 

Do you want something else tested here?

**Comment by urapadmin** [comment]
no, just wanted to make sure that this is still as planned. Thank you.
